### PR TITLE
Fix database creation doc url in install process

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | This change fixes minor code issues I found while working on another bug.Capture du 2020-11-27 23-40-32 (1)
| Type?         |  improvement 
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A.
| How to test?  | N/A.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22147)

### CODE REVIEW

**Review by Arman-Hosseini — APPROVED** [review]

File: app/Resources/translations/default/Install.xlf [hunk 1]
```diff
@@ -497,8 +497,8 @@ Comment: We fill the process step used for Ajax queries</note>
         <note>Line: 30</note>
       </trans-unit>
       <trans-unit id="74987401b3b33cc9542cd8585c08de98">
-        <source><![CDATA[To use PrestaShop, you must <a href="http://doc.prestashop.com/display/PS16/Installing+PrestaShop#InstallingPrestaShop-Creatingadatabaseforyourshop" target="_blank">create a database</a> to collect all of your store's data-related activities.]]></source>
-        <target><![CDATA[To use PrestaShop, you must <a href="http://doc.prestashop.com/display/PS16/Installing+PrestaShop#InstallingPrestaShop-Creatingadatabaseforyourshop" target="_blank">create a database</a> to collect all of your store's data-related activities.]]></target>
+        <source><![CDATA[To use PrestaShop, you must <a href="https://devdocs.prestashop.com/1.7/basics/installation/localhost/#creating-a-database-for-your-shop" target="_blank">create a database</a> to collect all of your store's data-related activities.]]></source>
```
Comment (matks) [inline app/Resources/translations/default/Install.xlf:500]: Indeed this link is old but I think the right link is rather https://doc.prestashop.com/display/PS17/Installing+PrestaShop#InstallingPrestaShop-Creatingadatabaseforyourshop

devdocs.prestashop.com is the developer documentation while doc.prestashop.com is the user (merchant) documentation. The installer aims for users I believe 🤔 (@LouiseBonnard please tell me if I got it wrong)

**Review by Progi1984 — CHANGES_REQUESTED** [review]
@PululuK Could you extract the link from the translated string ? (sprint use)

**Comment by matks** [comment]
Hello @PululuK what do we do about this PR 😉  ?

**Comment by PierreRambaud** [comment]
Hi @PululuK ,

Since we had no news from you for more than 30 days, I'll close this pull request. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**